### ISSUE TITLE
Want to Add 5 NFTs to the metadata.

### ISSUE THREAD

**Issue by granthughes1999** [HashLips/hashlips_art_engine#944]
Hello,

I want to add 5 custom NFTs and their associated metadata to the start of the collection. My plan is to run node index.js and then add the images and their metadata to the correct folders.

Is there a way to have node index.js start the first NFT and its associated metadata with the number 6? that way I can easily place the first 5 into the collection.

Thanks!

**Comment by granthughes1999** [comment]
SOLUTION

I am simply going to have the const shuffleLayerConfigurations = true; at which point I will render the full collection. and replace the first 5 images and metadata.

**Comment by thepeanutgalleryandco** [comment]
@granthughes1999 you can always have 6 layerconfiguration items. Then specify your unique items as growth size of 1 and then your last configuration as your less random editions.

You can also check out my repo where I have added filtration options - https://github.com/thepeanutgalleryandco/create-and-mint-nft-collection

**Comment by emirhansirkeci** [comment]
@thepeanutgalleryandco can you check this out too? https://github.com/HashLips/hashlips_art_engine/issues/942 is your repo able to do that?

**Comment by thepeanutgalleryandco** [comment]
@justChargin Yes, you can do both in my repo. I have answered your question with exclusions in the other ticket.

The below setup would make unique items for NFT for @granthughes1999  editions 1, 2, 3, 4 and 5, while 6 -20 (the next 15) will be randomised / generated between the different layers. You will see that for items 2, 3, 4, 5, 6 - 20, I removed certain layers that can be found in edition 1.

const layerConfigurations = [
  {
    growEditionSizeTo: 1,
    layersOrder: [
      { name: "Background" },
      { name: "Eyeball" },
      { name: "Eye color" },
      { name: "Iris" },
      { name: "Shine" },
      { name: "Bottom lid" },
      { name: "Top lid" },
    ]
  },
  {
    growEditionSizeTo: 2,
    layersOrder: [
      { name: "Background" },
      { name: "Eye color" },
      { name: "Iris" },
      { name: "Shine" },
      { name: "Bottom lid" },
      { name: "Top lid" },
    ]
  },
  {
    growEditionSizeTo: 3,
    layersOrder: [
      { name: "Background" },
      { name: "Eyeball" },
      { name: "Iris" },
      { name: "Shine" },
      { name: "Bottom lid" },
      { name: "Top lid" },
    ]
  },
  {
    growEditionSizeTo: 4,
    layersOrder: [
      { name: "Background" },
      { name: "Eyeball" },
      { name: "Eye color" },
      { name: "Shine" },
      { name: "Bottom lid" },
      { name: "Top lid" },
    ]
  },
  {
    growEditionSizeTo: 5,
    layersOrder: [
      { name: "Background" },
      { name: "Eyeball" },
      { name: "Eye color" },
      { name: "Iris" },
      { name: "Bottom lid" },
      { name: "Top lid" },
    ]
  },
  {
    growEditionSizeTo: 20,
    layersOrder: [
      { name: "Background" },
      { name: "Eyeball" },
      { name: "Eye color" },
      { name: "Iris" },
      { name: "Top lid" },
    ]
  }
];

**Comment by 3alola** [comment]
> 

this is soooo helpfull
